### DISCUSSION TITLE
Is it possible to copy issue from original repo?

### DISCUSSION THREAD

**Discussion by drakon660** [tye-v2/tye2#11] [Ideas]
Can we copy somehow original issues?

**Comment by razvangoga** [comment]
I dont's think so - i made this repo from scratch - it's not a fork of the original one (in the github UI ideea of fork).

If you think some issue from there is important - just open a new one and link to the original one

**Comment by drakon660** [comment]
I will go through all those issues and check if there is some stuff needed consideration or checking.

**Reply by razvangoga** [reply]
Sure 👍 

I've already merged all the open PRs that made sense to have (see the list in #1) that either fix or enhance the local dev experience.
If i misses anything link it here and let's discuss it.

**Reply by drakon660** [reply]
I know you did it, maybe among all those issues will be  a bug, or inspiration, as I sad there is a lot to refactor, but lets go with small steps, please check my comment regarding release v1.

**Comment by drakon660** [comment]
closing
